lib/chain_sync.ex

```diff
@@ -90,7 +90,15 @@ defmodule Xander.ChainSync do
       state: state
     }
 
-    :gen_statem.start_link({:local, __MODULE__}, __MODULE__, data, [])
+    if sync_from && sync_from == :origin && network != :yaci_devkit do
```

Comment:
What's the difference between syncing from a standard node from origin and syncing from yaci with `create node --era conway`?